### DISCUSSION TITLE
Use @hocuspocus/server on client with sqlite-wasm and OPFS

### DISCUSSION THREAD

**Discussion by dpschen** [ueberdosis/hocuspocus#564] [Q&A] [answered]
Hello 👋

Can I run @hocuspocus/server offline on my client, so that I can save the data via [SQLite/wasm](https://sqlite.org/wasm/) in the [origin private file system (OPFS)](https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API#origin_private_file_system)?

**Comment by janthurau** [comment, marked as answer]
hey @dpschen, yep that should work.

**Reply by dpschen** [reply]
> hey @dpschen, yep that should work.

Wow that's nice! Thanks for the quick answer.
Let's see if I can find some time to fiddle around with that.
